internal/api/design/package_.go

```diff
@@ -249,7 +249,7 @@ var _ = Service("package", func() {
 		Error("internal_error", ErrorResult, "Fault while processing upload.")
 
 		HTTP(func() {
-			POST("/upload")
+			POST("/sip/upload")
```

Comment:
I'm not clear on the intended semantics here.  Generally I think the "/sips" endpoint represents data *about* a SIP, rather then the actual SIP.  If the intention is for a single endpoint to create a SIP data record AND upload the SIP then I think `POST /sips` is sufficient.  If the SIP data is created separately from the upload of the actual SIP then I think  `POST /sips/{id}/upload" is more appropriate.

Reply:
Let me know if you feel strong about using `POST /sips`, it should be quick to change. Otherwise, I think it's okay as it is.